README.adoc

```diff
@@ -247,7 +240,7 @@ validate the health of the inventory manager application with the `system` and
 `inventory` services.
 
 Begin by creating health check test class for the two services
-in the `/src/test/java/it/io/openliberty/guides/health/HealthTest.java` file:
+in the `src/test/java/it/io/openliberty/guides/health/HealthTest.java` file:
```

Comment:
Begin by creating [a] health check test class for the two services in the `src/test/java/it/io/openliberty/guides/health/HealthTest.java` file:

Reply:
fixed